tests/unit/gapic/documentai_v1beta3/quickstart_v1beta3_test.py

```diff
@@ -0,0 +1,70 @@
+# # Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from uuid import uuid4
+import pytest
+import sys, os
+from google.cloud import storage
+import google.api_core as api_core
+
+from samples import quickstart_sample_v1beta3
+
+from google.cloud.documentai_v1beta3.services.document_processor_service import (
+    DocumentProcessorServiceClient,
+)
+from google.cloud.documentai_v1beta3.services.document_processor_service import (
+    transports,
+)
+from google.cloud.documentai_v1beta3.types import document
+from google.cloud.documentai_v1beta3.types import document_processor_service
+from google.cloud.documentai_v1beta3.types import geometry
+from google.cloud.documentai_v1beta3.types import ProcessRequest
+
+
+location = "us"
+project_id = '1012616486416'
```

Comment:
I think we might want to get the `project_id` from environment variables.
```
project_id = os.environs['GCLOUD_PROJECT']
```

There may be a niftier way to get this from the Document AI client (but I don't know it off the top of my head).